src/main/js/constants/TableauxConstants.js

```diff
@@ -117,6 +118,11 @@ var TableauxConstant = {
     formatForUser: "DD.MM.YYYY - HH:mm"
   },
 
+  DateFormats: {
+    formatForServer: "YYYY-MM-DD",
+    formatForUser: "DD.MM.YYYY"
```

Comment:
Can `formatForUser` be localized somehow?
This is just a question and should be tracked, if we want / need that. American users may want to have MM/DD/YYYY or something...